tools/make_dictionary_file.py

```diff
@@ -0,0 +1,65 @@
+"""
+Makes dictionary file from mozc
+(https://github.com/google/mozc/tree/master/src/data/dictionary_oss)
+"""
+
+def make_jp_name_dictionary(input_file_names):
+    """
+    Makes japanese name dictionary.
+
+    Args:
+        input_file_names: a list of file names
+
+    Out put format:
+        kanji '\t' yomigana(hiragana) '\n'
+        kanji '\t' yomigana(hiragana) '\n' ...
+    """
+
+    yomigana_list = []
+    for input_file_name in input_file_names:
+
+        with open(input_file_name, 'r') as f:
```

Comment:
Why do you prefer to avoid "with"? It looks rather preferred to use:
https://google-styleguide.googlecode.com/svn/trunk/pyguide.html?showone=Files_and_Sockets#Files_and_Sockets